app/src/Containers/Login-Logout/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { LoginOutButton } from "./styles"
+import React from "react"
+import { useSelector, useDispatch } from "react-redux"
+import { logoutUser } from "../../actions"
+
+const LoginOut = (props: any) => {
+  const isLoggedIn = useSelector(state => state.user.isVerified)
+  const dispatch = useDispatch()
+
+  const handleButton = () => {
+    if(!isLoggedIn) {
+      props.showForm()
+    }
+    else{
+      dispatch(logoutUser())
+    }
+  }
```

Comment:
u should just have tow different handlers for logout/login and button with distinct actions b/c these *always* get huge eventually